com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/CreateOrUpdateAppStep.java

```diff
@@ -245,6 +251,14 @@ public void handleApplicationAttributes() {
 
         @Override
         public void handleApplicationServices() throws FileStorageException {
+            Boolean passedTestingPhase = (Boolean) execution.getContext()
+                                                            .getVariable(Constants.VAR_PASSED_TESTING_PHASE);
+            if (processTypeParser.getProcessType(execution.getContext())

```

Comment:
Does it matter whether the deployment is "blue-green" or not? If the variable is set to `true` we should skip the bindings.